### PR TITLE
add README and LICENSE file

### PR DESCRIPTION
**Pull request by dhmlau**
Add basic files for the repo:
- update README
- add LICENSE
- add the issue template

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Copyright (c) IBM Corp. 2014,2017. All Rights Reserved.
```
Comment (shimks) [inline LICENSE]: 2018?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 # v4.loopback.io
-LoopBack 4 Web Site
+![LoopBack 4 logo](http://loopback.io/images/branding/mark/blue/loopback.jpg width=100)
```
Comment (shimks) [inline README.md]: When I click `View` when reviewing, there's no image showing up. Will it work once it's merged?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,7 @@
 # v4.loopback.io
-LoopBack 4 Web Site
+![LoopBack 4 logo](http://loopback.io/images/branding/mark/blue/loopback.jpg | width=100)
```
Comment (shimks) [inline README.md]: When I preview this, I don't get the image. Is this the way it's supposed to be?

File: README.md (hunk 2 above)
Comment (dhmlau) [reply README.md]: i've just fixed it :)

**Pull request merged**